fab/fab/operations/db.py

```diff
@@ -93,6 +93,7 @@ def migrate():
     with cd(env.code_root):
         sudo('%(virtualenv_root)s/bin/python manage.py sync_finish_couchdb_hq' % env)
         sudo('%(virtualenv_root)s/bin/python manage.py migrate_multi --noinput' % env)
+        sudo('%(virtualenv_root)s/bin/python manage.py check --deploy' % env)
```

Comment:
I'm not sure `migrate` is the appropriate place for this, as this could end up checking a bunch of stuff.

Reply:
Yea, agree. I'll move it.

Reply:
021a381